x/emissions/module/rewards/rewards.go

```diff
@@ -455,9 +462,18 @@ func payoutRewards(
 				))
 				continue
 			}
+
+			if reward.Type == types.WorkerInferenceRewardType {
+				infererRewards = append(infererRewards, reward)
+			} else if reward.Type == types.WorkerForecastRewardType {
+				forecasterRewards = append(forecasterRewards, reward)
+			}
 		}
 	}
 
+	types.EmitNewInfererRewardsSetEvent(ctx, blockHeight, infererRewards)
+	types.EmitNewForecasterRewardsSetEvent(ctx, blockHeight, forecasterRewards)
+	types.EmitNewReputererAndDelegaterRewardsSetEvent(ctx, blockHeight, reputerAndDelegatorRewards)
```

Comment:
same. Maybe just don't call them "set" events?

Reply:
renamed: settled

Reply:
There's a spelling error in reputer